index.js

```diff
@@ -1,5 +1,3 @@
-const fetchCsv = require('./lib/fetch')
+import fetchCsv from './fetch.js'
```

Comment:
I'd just reexport, like

```suggestion
export { fetchCsv as fetch } from './fetch.js'
```

Reply:
I think you mean `export { default as fetch } from './fetch.js'`?

Reply:
right, right